flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -222,6 +223,19 @@ public class RestartStrategyOptions {
                                             code(RESTART_STRATEGY.key()), code("exponential-delay"))
                                     .build());
 
+    @Documentation.OverrideDefault("infinite")
+    public static final ConfigOption<Integer> RESTART_STRATEGY_EXPONENTIAL_DELAY_ATTEMPTS =
+            ConfigOptions.key("restart-strategy.exponential-delay.attempts-before-reset-backoff")
+                    .intType()
+                    .defaultValue(Integer.MAX_VALUE)
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "The number of times that Flink retries the execution before the job is declared as failed "
+                                                    + "before reset the backoff to its initial value if %s has been set to %s.",
```

Comment:
`reset` -> `resetting`

Reply:
Thanks @RocMarshal for the review! Updated~